ci/bamboo/bootstrap-manylinux-prototype.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -o errexit
+set -o xtrace
+
+NUPIC_CORE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../.." && pwd )"
+
+#echo "NUPIC_CORE_DIR=${NUPIC_CORE_DIR}"
+
+DOCKER_IMAGE="quay.io/numenta/manylinux1_x86_64_centos6"
```

Comment:
@scottpurdy, consider that when we're able to switch to the stock manylinux docker image, we won't have image versions, since they don't provide them. Is there any point in having us introduce versions into our private manylinux docker images knowing that we'll give that up when switching to the community one?